cmd/create/operatorroles/cmd.go

```diff
@@ -102,10 +102,22 @@ func modeCompletion(cmd *cobra.Command, args []string, toComplete string) ([]str
 	return modes, cobra.ShellCompDirectiveDefault
 }
 
-func run(cmd *cobra.Command, _ []string) {
+func run(cmd *cobra.Command, argv []string) {
 	reporter := rprtr.CreateReporterOrExit()
 	logger := logging.CreateLoggerOrExit(reporter)
 
+	// Allow the command to be called programmatically
+	skipInteractive := false
+	if len(argv) == 3 && !cmd.Flag("cluster").Changed {
+		args.clusterKey = argv[0]
+		args.mode = argv[1]
+		args.permissionsBoundary = argv[2]
+
+		if args.mode != "" {
+			skipInteractive = true
```

Comment:
It just occurred to me that if the user explicitly sets _both_ `--interactive` and `--mode`, we might want to actually display the interactive mode here. What do you think?